src/templates/app/user/authenticate.jsx

```diff
@@ -94,6 +97,17 @@ const Authenticate = () => (
                             <img className='gr-padding-20' src={it.url_for('images/pages/authenticate/valid.svg')} />
                             <h1 className='gr-padding-10'>{it.L('Your proof of identity has been verified successfully')}</h1>
                         </div>
+
+                        <div id='personal_details_error' className='center-text gr-padding-20 invisible'>
+                            <h2 className='gr-padding-10'>{it.L('Update your personal details')}</h2>
+                            <p>{it.L('We can\'t validate your personal details because there is some information missing.\nPlease update your telephone to continue.')}</p>
```

Comment:
please don't use `\n` in the message. use two paragraph tags instead.

Reply:
and we are not hardcoding `telephone`. you can take that from the API error details field and concatenate them with a comma here. look at how cashier is doing it.